src/ERC1155Mappings/premintMappings.ts

```diff
@@ -18,3 +18,21 @@ export function handlePreminted(event: Preminted): void {
 
```

Comment:
it looks like we can add `premint.quantityMinted = event.params.quantityMinted;` here also since it exists in the original event